styles.css

```diff
@@ -0,0 +1,673 @@
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+html {
+  scroll-behavior: smooth;
+}
+
+body {
+  background-color: #fff;
+  font-family: 'PT Sans', sans-serif;
+}
+
+a {
+  color: black;
+  text-decoration: none;
+}
+
+h2 {
+  color: #ff451f;
+  font-size: 150%;
+  padding: 8px;
+}
+
+.hide-all {
+  display: none;
+}
+
+.strike {
+  text-decoration: line-through;
+}
+
+/* <!-- HEADER SECTION --> */
+
+header {
+  background-color: #fff;
+  height: 46px;
+  position: sticky;
+  top: 0;
+  z-index: 101;
+}
+
+.right-side img {
+  position: absolute;
+  padding: 5px 8px;
+  border-left: 1px solid rgb(235, 235, 235);
+}
+
+.menu-icon {
+  position: absolute;
+  bottom: 0;
+  right: 0;
+}
+
+.search-icon {
+  top: -6px;
+  right: 52px;
+}
+
+/* <!-- MAIN SECTION --> */
+
+#main-section {
```

Comment:
- [ ] Avoid using id selectors to style elements in CSS. [Check this article with reasons for why that's a bac practice](https://dev.to/clairecodes/reasons-not-to-use-ids-in-css-4ni4) and change the selectors to class selectors(you can always [combine selectors to be have more specificity](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity) if that was the reason for using ids).

Reply:
Thank you @uryelah 

Very Professional, descriptive, and clean code review ⭐⭐⭐📝

We are working on it and would push it ASAP...

@od-c0d3r
@Zappat0n